views/partials/board.jade

```diff
@@ -1,8 +1,6 @@
-.video-wrapper(ng-if='!game.round')
```

Comment:
Is there a reason why the intro video was removed?
I understand that the original video (https://www.youtube.com/embed/eNkvxqGmscY?autoplay=1&loop=1&playlist=eNkvxqGmscY) is not available anymore.
It might be a nice idea to show another/similar into the video (we liked it in our game)

Reply:
I actually have an MP4 that we could add to the repo to show instead, but not sure if this would have any issues from a copyright standpoint?  We can create a separate commit to do this if that would be of interest to you.

Reply:
Good point. I am not sure.
Let's have it removed for now. It is good enough :)